core/templates/dev/head/pages/landing/topic_landing_page.html

```diff
@@ -1,7 +1,7 @@
 {% extends 'pages/base.html' %}
 
 {% block maintitle %}
-  Fractions
+  Learn {{ title }} - Oppia
```

Comment:
Could you investigate whether there is a way to retrieve this via AJAX instead? We are trying to remove the use of Jinja from the codebase and it seems prudent not to add to it instead.

Reply:
Done!